django/db/models/base.py

```diff
@@ -519,11 +519,14 @@ def __hash__(self):
         return hash(self.pk)
 
     def __reduce__(self):
-        data = self.__dict__
+        data = self.__getstate__()
         data[DJANGO_VERSION_PICKLE_KEY] = get_version()
         class_id = self._meta.app_label, self._meta.object_name
         return model_unpickle, (class_id,), data
 
+    def __getstate__(self):
+        return self.__dict__.copy()
```

Comment:
I'm not sure about the decision but either way, perhaps the concerns should be documented in the "Pickling objects" section of `docs/ref/models/instances.txt`.